### PR TITLE
#5581 Enable users to configure the compression type used by source c…

### LINKED ISSUE #5581: Enable users to configure the compression type used by source connector
expose ```compression.type``` to users

### PR DESCRIPTION
**Pull request by chia7712**
…onnector

### Checklist

- [ ] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [ ] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [ ] Add an assignee (of course, you are the assignee by default)
- [ ] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [ ] Link to an issue fix #5581
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-client/src/main/scala/oharastream/ohara/client/configurator/WorkerApi.scala [hunk 1]
```diff
@@ -54,6 +58,8 @@ object WorkerApi {
   val OFFSET_TOPIC_PARTITIONS_KEY   = "offset.storage.partitions"
   val OFFSET_TOPIC_REPLICATIONS_KEY = "offset.storage.replication.factor"
 
+  val COMPRESSION_TYPE_KEY: String = "producer." + ProducerConfig.COMPRESSION_TYPE_CONFIG
```
Comment (jackyoh) [inline ohara-client/src/main/scala/oharastream/ohara/client/configurator/WorkerApi.scala:61]: 是否考慮用 s"producer.${ProducerConfig.COMPRESSION_TYPE_CONFIG}"

**Review by jackyoh — APPROVED** [review]

File: ohara-client/src/main/scala/oharastream/ohara/client/configurator/WorkerApi.scala (hunk 1 above)
Comment (chia7712) [reply ohara-client/src/main/scala/oharastream/ohara/client/configurator/WorkerApi.scala:61]: good point. will copy that

**Comment by chia7712** [comment]
retry configurator

**Comment by chia7712** [comment]
retry streamIT

**Pull request merged**